code/center.m

```diff
@@ -7,4 +7,4 @@
     %   Returns a new array containing the values in
     %   DATA centered around the value.
 
-    out = (data  - mean(data)) + desired;
+    out = (data  - mean(data(:))) + desired;
```

Comment:
I think we decided to leave the function unchanged, and instead document that it won't work for matrices. This *will* work for matrices..